airbyte-integrations/connectors/source-shopify/integration_tests/configured_catalog.json

```diff
@@ -336,6 +336,18 @@
       "cursor_field": ["created_at"],
       "destination_sync_mode": "append"
     },
+    {
```

Comment:
I got this error when running a sync with this configured_catalog
`'The stream articles no longer exists in the configuration. Refresh the schema in replication settings and remove this stream from future sync attempts.'
`

Reply:
Because we have the check for `scopes`  while generating the list of available streams, prior to the availability strategy. Why would we need to list a stream that is not available for the authenticated user based on his scopes granted and make empty checks on top just to say: `Oops, you've got no permissions to read this stream, no matter if it's listed for you in the UI`  - confusing.